BUIP-HF/replay-protected-sighash.md

```diff
@@ -0,0 +1,191 @@
+# BUIP-HF Digest for replay protected signature verification accross hard forks
+
+## Abstract
+
+This document describes proposed requirements and design for a reusable signing mechanism ensuring replay protection in the event of a hard fork. It provides a way for users to create transactions which are only valid on one branch of the fork.
+
+The proposed digest algorithm is similar to [BIP143](https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki) as it is recognized as a superior solution to the currently used algorithm [1] and is already implemented in a wide variety of applications.
```

Comment:
I feel it would be nice to add in brief summary of the benefits here too, notably this part from "Abstract" of BIP143. 

> minimize redundant data hashing in verification, and to cover the input value by the signature. 

That really makes some key benefits clear whereas the linked Motivation section is complex...